profiles/views.py

```diff
@@ -96,33 +95,47 @@ def get_queryset(self, pk=None):
         if self.request.method in ['PUT', 'PATCH']:
             return Profile.objects.filter(profile_id=pk)
 
+    def get_serializer_class(self):
+        get_contacts = self.request.query_params.get("get_contacts")
```

Comment:
I suggest to rename it a bit. `get_contacts` → `with_contacts`.

Reply:
Done. Thanks :)